tests/sentry/api/endpoints/relocations/test_index.py

```diff
@@ -104,6 +106,13 @@ def setUp(self):
 
         self.success_uuid = Relocation.objects.get(status=Relocation.Status.SUCCESS.value)
 
+    @with_feature("auth:enterprise-staff-cookie")
+    def test_good_staff_simple(self):
+        self.login_as(user=self.staff_user, staff=True)
```

Comment:
I know with superuser, it is possible for a user to be `is_superuser=True`, but `superuser=False`, meaning that the user has superuser privileges but is not currently authenticated for them. Does the same distinction exist for staff as well now?

Reply:
Yes you can have the same for staff, where you are a staff user but don't login with active staff